replicate/files.py

```diff
@@ -2,10 +2,136 @@
 import io
 import mimetypes
 import os
-from typing import Optional
+from typing import Any, Dict, List, Optional
 
 import httpx
 
+from replicate.resource import Namespace, Resource
+
+
+class File(Resource):
+    """A file on the server."""
+
+    id: str
+    """The ID of the file."""
+
+    name: str
+    """The name of the file."""
+
+    content_type: str
+    """The content type of the file."""
+
+    size: int
+    """The size of the file in bytes."""
+
+    etag: str
+    """The ETag of the file."""
+
+    checksum: str
+    """The checksum of the file."""
+
+    metadata: Dict[str, Any]
+    """The metadata of the file."""
+
+    created_at: str
+    """The time the file was created."""
+
+    expires_at: Optional[str]
+    """The time the file will expire."""
+
+    urls: Dict[str, str]
+    """The URLs of the file."""
+
+
+class Files(Namespace):
+    def create(
+        self, file: io.IOBase, metadata: Optional[Dict[str, Any]] = None
+    ) -> File:
+        """Create a file on the server."""
```

Comment:
What does "server" mean in this context? I wonder if there's a clearer way to word this, like "Create and upload a file to a remote host" or something?